src/vm/ClrEtwAll.man

```diff
@@ -3573,6 +3610,11 @@
                            task="AssemblyLoader"
                            symbol="AssemblyLoadFromResolveHandlerInvoked" message="$(string.RuntimePublisher.AssemblyLoadFromResolveHandlerInvokedEventMessage)"/>
 
+                    <event value="296" version="0" level="win:Informational"  template="ResolutionAttempted"
+                           keywords ="AssemblyLoaderKeyword" opcode="win:Resolve"
```

Comment:
We should define a custom opcode under the `AssemblyLoader` task for this event.